src/components/FileInputMetadata.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import filesize from 'filesize';
+import { string, number } from 'prop-types';
+import '../styles/FileInputMetadata.scss';
+
+const styleName = name => {
+  // Do we have list of supported image types ??
+  const imageTypes = ['.jpg', '.jpeg', '.png', '.gif'];
+  const typeIndex = imageTypes.map(type => name.indexOf(type));
+
+  return {
+    name: name.slice(0, typeIndex[0]),
+    type: name.slice(typeIndex[0]),
+  };
+};
+
+const FileInputMetadata = props => {
+  const { name, type } = styleName(props.name);
+  return (<div className="brainhubFileInput__metadata">
+    <div className="brainhubFileInput__metadata__image">
+
+      {/* This is placeholder for now */}
```

Comment:
Just friendly protip: we have a handy rule for eslint. When you start your comment with word `TODO:` then it will find all these comments and warn about it. So in the future, it will be easier to spot temporary solutions and fix it.